tidb-cloud/terraform/terraform-overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: TiDB Cloud Terraform Provider Overview
+summary: Create, manage, and update your TiDB Cloud resources through Terraform
+---
+
+# TiDB Cloud Terraform Provider Overview
+
+As a fully managed service of TiDB, TiDB Cloud can automate database maintenance operations. With the [TiDB Cloud API](https://docs.pingcap.com/tidbcloud/api/v1beta), you can programmatically handle operations such as deployment, scaling, and restore.
+
+If you are looking for a simplified way to automate resource provisioning and your infrastructure workflow. You can try out our [TiDB Cloud Terraform Provider](https://registry.terraform.io/providers/tidbcloud/tidbcloud), which provides you with the following capabilities:
```

Comment:
Could you also provide a few sentences to let users know the following information? 

- What is Terraform (one sentence to describe it)
- What is TiDB Cloud Terraform Provider here?  (one sentence to describe it)
- What is the relationship between TiDB Cloud API and TiDB Cloud Terraform Provider?

For example, https://docs.pingcap.com/tidb/stable/clinic-introduction